source/mongorestore/mongorestore-examples.txt

```diff
@@ -345,14 +345,22 @@ namespace:
        You cannot restore the ``system.buckets.weather.bson`` file
        by itself. Attempting to do so results in an error.
 
+Back Up and Restore a Database to a Specific Point in Time
```

Comment:
[non-blocking suggestion]

I think we should consider using "**at** a specific point" throughout. 

Spitballing:

"To back up a database at a specific point in time to an :term:`oplog` file...".

Reply:
Done.